test/SaveHandler/MongoDBTest.php

```diff
@@ -69,6 +76,20 @@ public function tearDown()
         }
     }
 
+    public function testConstructorThrowsException()
+    {
+        $notMongo = new \stdClass();
+
+        $expectedExceptionMessage =  sprintf(
+            'Parameter of type %s is invalid; must be MongoClient or Mongo',
+            get_class($notMongo)
```

Comment:
Expectation should be static and not be built with the same logic as the implementation